### DISCUSSION TITLE
[Question][Hw] About updating previous HW

### DISCUSSION THREAD

**Discussion by ee12ha0220** [prosyslab-classroom/cs348-information-security#559] [Q&A]
HW4를 하는 과정에서 instrument.ml을 조금 수정했는데, 이를 HW3에 update해도 되나요? 변수 이름 같은 것을 조금 고치고 utils.ml에 있는 함수들을 이용해서 code를 조금 보기 쉽게 바꾼 것인데, 그냥 나중에 혼자 보는 것을 생각해서 update를 하고 싶습니다. HW3채점을 할 때 기한 내에 commit한 것을 기준으로 채점이 되나요? 아니면 이렇게 나중에 수정을 했을 때 늦게 commit한 것으로 간주되어서 점수에 감점이 있나요? 감사합니다 :)

**Comment by KihongHeo** [comment]
I appreciate that you try to improve your code further.
Can you push your code after TAs finish the grading?
It will be released soon. After that you can do whatever you want.

**Comment by ee12ha0220** [comment]
Thank you ! :)
